drake/bindings/python/pydrake/test/testRBTCoM.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import print_function
```

Comment:
this code allows us to be agnostic (the same code will run correctly in python2 and python3. I've also verified that the basic capabilities work in python3, but stopped short of making it the default. I'm open to requiring python3, though. It might also solve the ridiculous path issues I'm having on the mac build servers.